pages/news/2020-02-20-mixxx-accepted-for-gsoc-2020.html

```diff
@@ -0,0 +1,24 @@
+title: Mixxx accepted for GSoC 2020!
+author: Jan Holthuis
+date: 2020-02-20 20:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Mixxx has been [accepted as a mentoring organization](https://summerofcode.withgoogle.com/organizations/6577142898360320/) for [Google Summer of Code 2020](https://summerofcode.withgoogle.com/), a global program that aims to bring more student developers into the world of open source development.
+
+This means that students have to opportunity to work on Mixxx full-time as a 3 month programming project and receive a stipend sponsored by Google. For more information, check out the [Google Summer of Code FAQ](https://developers.google.com/open-source/gsoc/faq#students).
```

Comment:
sorry there is still a typo:   "This means that students have *the* opportunity"